components/Edition.vue

```diff
@@ -0,0 +1,16 @@
+<script lang="ts" setup></script>
+
+<template>
+  <div class="edition">
+    &mdash; First Edition, March 2024 &mdash;
+  </div>
```

Comment:
I like this a lot! It makes it loud and clear when there's new stuff to see in ARDAC Explorer.